docs/installation.rst

```diff
@@ -64,19 +64,14 @@ meaning that it must be compiled on your own machine. But, luckily, you won't ha
 Simply install the build dependencies on your machine and the build process itself will happen behind
 the scenes when you install the package.
 
-If you are on Fedora, install the build dependencies with this command:
+Caveat: Unfortunately, a fully-featured ``createrepo_c`` can only be built on Red Hat based distros,
```

Comment:
@dralley, I'd mention CentOS and Fedora explicitly. What do you think?

Reply:
@goosemania Done